app/src/lib/dispatcher/dispatcher.ts

```diff
@@ -806,6 +806,17 @@ export class Dispatcher {
         } catch (e) {
           rejectOAuthRequest(e)
         }
+
+        // maybe a workaround for macOS not focusing on the app after
+        // completing the OAuth dance?
+        const window = remote.getCurrentWindow()
+        const isFocused = window.isFocused()
+        log.debug(`OAuth dance done, isFocused: ${isFocused}`)
```

Comment:
I'm still pondering on this overall fix.

It'd be nice to see if someone can find an entry that looks like this in their log when running the test build:

```
debug: [ui] OAuth dance done, isFocused: false
```

This means that the app is not focused after we've completed the OAuth flow - which fits with the issue - and the fix below is the right thing to do.

@tierninho had an entry that was close:

```
debug: [ui] OAuth dance done, isFocused: true
```

This means the fix below is unnecessary, but that might be because we weren't able to trigger the problem.

Alternatively, this change is relatively minor, and is probably safe to keep around if it's a rare occurrence.